src/utils/Utils.ts

```diff
@@ -487,6 +487,17 @@ export default class Utils {
     return 0;
   }
 
+  public static convertWattHourToKiloWattHour(wattHours: number, decimalPlaces?: number): number {
+    if (decimalPlaces) {
+      return Utils.roundTo((wattHours / 1000), decimalPlaces);
```

Comment:
Use truncTo() instead to avoid prices with not really consumed energy.

Reply:
ok, changed